pages/_app.js

```diff
@@ -49,6 +49,35 @@ export default function MyApp({ Component, pageProps }) {
           <Component {...pageProps} />
         </ThemeProvider>
       </StyledEngineProvider>
+      <style jsx global>{`
```

Comment:
I suggest to remove this commit for now as it is not 100% working (some buttons are hidden under address bar, pull-to-refresh is enabled in PWA)